src/build/util.py

```diff
@@ -0,0 +1,64 @@
+# SPDX-License-Identifier: MIT
+
+import os
+import pathlib
+import sys
+import tempfile
+import typing
+
+from typing import Union
+
+import pep517
+
+import build
+import build.env
+
+
+if sys.version_info >= (3, 8):
+    import importlib.metadata as importlib_metadata
+else:
+    import importlib_metadata
+
+
+if typing.TYPE_CHECKING:
+    from importlib_metadata import PackageMetadata, SimplePath
+
+
+def _project_wheel_metadata(builder: build.ProjectBuilder) -> 'PackageMetadata':
```

Comment:
Does it return `PackageMetadata` if you don't have `importlib-metadata >= 4.1.0`?